manim/animation/transform.py

```diff
@@ -71,7 +73,9 @@ def path_arc(self) -> float:
     @path_arc.setter
     def path_arc(self, path_arc: float) -> None:
         self._path_arc = path_arc
-        self._path_func = path_along_arc(self._path_arc, self.path_arc_axis)
+        self._path_func = path_along_arc(
+            self._path_arc, self.path_arc_axis, self.path_arc_centers
```

Comment:
I have a (slight) preference for passing keyword arguments as actual keyword arguments.

```suggestion
            self._path_arc,
            axis=self.path_arc_axis,
            arc_centers=self.path_arc_centers
```